### ISSUE TITLE
When reaching the maximum number of training data will stop improving training accuracy

### ISSUE THREAD

**Issue by corentin87** [AlexeyAB/darknet#3115]
Hi,

I guess this is more a general question about training a CNN but YOLO is the one i'm using. I've started my training set for 'person' detections by labelling some data from different cameras videos (in similar environment).... Every time I was adding new data for a new camera I was retraining YOLO, which actually improved the detection for this camera. For the training, I split my data randomly into training/validation set. I use the validation set to compute accuracy. This is not overfitting as all the previous data are also used in the training. 

Now, I've gathered more than 100 000 labelled data. I was expecting to not have to train anymore at this point as my data set is pretty big. But looks like I still need to do it. if i'm getting a new camera video, labelling 500-1000 samples, adding them to my huge data set and training again, the accuracy is improving for this camera.
I don't understand really understand why. Why do i still need to add new data to my set? Why is the accuracy improving a lot on the new data, while there are 'drawn' in the thousands of already existing data? Is there a point where I will be able to stop training because adding new data will not improve the accuracy?

Thanks for sharing your thoughts and ideas!

**Comment by corentin87** [comment]
thanks for your answer Alex. My training set for one class includes 21 000 images with 170 000 bounding boxes. They are all taken from similar environment (fisheye, indoor,..).So I believe that new images are already similar to some images. 
Maybe i didn't go deep enough in the the understanding of CNN, but by adding 100 new images to a pool of 21 000 previous images for the training (which is a small portion), i'm getting much better accuracy on the new videos. Does this sounds right to you? The fact that the network can learn new features while he add already a lot to learn with.
Thanks

**Comment by AlexeyAB** [comment]
It means that new images are very different from old images.
